en/docs/learn/password-recovery.md

```diff
@@ -210,21 +211,21 @@ To try this out, first create a user in the Identity Server.
     ![add-new-user](../assets/img/using-wso2-identity-server/add-new-user.png) 
 
 4.  Log out of the Identity Server.
-5.  The URL for accessing dashboard is the following if the hostname is
+5.  The URL for accessing user portal is the following if the hostname is
     localhost and the Identity Server is running on port 9443:
-    <https://localhost:9443/dashboard/>. Click this link to access the
-    dashboard and log in using the credentials of the user you just
+    <https://localhost:9443/user-portal/>. Click this link to access the
+    user portal and log in using the credentials of the user you just
     created.  
     ![enter-using-credentials](../assets/img/using-wso2-identity-server/enter-using-credentials.png)   
-6.  Click the **View Details** button under the **Account Recovery**
-    section in the end user dashboard.  
-    ![view-details-account-recovery](../assets/img/using-wso2-identity-server/view-details-account-recovery.png) 
+6.  Go to the **Security** tab. Click the **+** button near **Security questions** under the
+    **Account Recovery** section.  
+    ![view-details-account-recovery](../assets/img/using-wso2-identity-server/security-account-recovery.png) 
 7.  Set challenge questions for the user account. There are two sets of
     challenge questions by default. You can pick one question for each
     set and give an answer for the question.  
     ![answer-challenge-questions](../assets/img/using-wso2-identity-server/answer-challenge-questions.png)   
-8.  Click **Update**.
-9.  Sign out of the dashboard and click **Forgot Password** on the login
+8.  Click **Save**.
+9.  Sign out of the user portal and click **Forgot Password** on the login
     screen.  
     ![choose-forgot-password](../assets/img/using-wso2-identity-server/choose-forgot-password.png) 
```

Comment:
The new image has a red box highlighting the word "Password". Can we adjust the box to make sure it's a clean line and doesn't cover the word "or" in that image?

Reply:
Done